notebooks/Grid Search for Binary Classification.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Grid Search for Binary Classification\n",
     "\n",
-    "This notebook goes through applying the `GridSearch` algorithm in FairLearn to a binary classification problem, where we also have a binary protected attribute. This algorithm comes from the paper [\"A Reductions Approach to Fair Classification\" (Agarwal et al. 2018)](https://arxiv.org/abs/1803.02453). The grid search is a simplified version of the full algorithm (appearing in section 3.4), which works best for binary classification and a binary protected attribute.\n",
+    "This notebook goes through applying the `GridSearch` algorithm in Fairlearn to a binary classification problem, where we also have a binary protected attribute. This algorithm comes from the paper [\"A Reductions Approach to Fair Classification\" (Agarwal et al. 2018)](https://arxiv.org/abs/1803.02453). The grid search is a simplified version of the full algorithm (appearing in section 3.4), which works best for binary classification and a binary protected attribute.\n",
```

Comment:
Not sure this quite belongs in this PR....

Reply:
I replaced all occurrences of FairLearn with Fairlearn. There was one outside of the documentation.